packages/mdc-theme/_variables.scss

```diff
@@ -69,18 +70,8 @@ $mdc-theme-property-values: (
   secondary: $mdc-theme-secondary,
   /* Background */
   background: $mdc-theme-background,
-  // Text-primary on "primary" background
-  text-primary-on-primary: mdc-theme-ink-color-for-fill_(primary, $mdc-theme-primary),
-  text-secondary-on-primary: mdc-theme-ink-color-for-fill_(secondary, $mdc-theme-primary),
-  text-hint-on-primary: mdc-theme-ink-color-for-fill_(hint, $mdc-theme-primary),
-  text-disabled-on-primary: mdc-theme-ink-color-for-fill_(disabled, $mdc-theme-primary),
-  text-icon-on-primary: mdc-theme-ink-color-for-fill_(icon, $mdc-theme-primary),
-  // Text-primary on "secondary" background
-  text-primary-on-secondary: mdc-theme-ink-color-for-fill_(primary, $mdc-theme-secondary),
-  text-secondary-on-secondary: mdc-theme-ink-color-for-fill_(secondary, $mdc-theme-secondary),
-  text-hint-on-secondary: mdc-theme-ink-color-for-fill_(hint, $mdc-theme-secondary),
-  text-disabled-on-secondary: mdc-theme-ink-color-for-fill_(disabled, $mdc-theme-secondary),
-  text-icon-on-secondary: mdc-theme-ink-color-for-fill_(icon, $mdc-theme-secondary),
+  on-primary: $mdc-theme-on-primary,
```

Comment:
Just to confirm... are we expecting to update/prune the rest of this map (and the readme) in other imminent PRs?